routers/repo/http.go

```diff
@@ -463,15 +465,22 @@ func serviceRPC(h serviceHandler, service string) {
 	// set this for allow pre-receive and post-receive execute
 	h.environ = append(h.environ, "SSH_ORIGINAL_COMMAND="+service)
 
+	// FIXME: graceful: Is the defaultContext appropriate here or should it be the requests context?
```

Comment:
>
> // FIXME: graceful: Is the defaultContext appropriate here or should it be the requests context?

I guess it depends on the operation. For a read operation (e.g. `clone`), as soon as the http request is gone the git command should die. For a writing operation that could be very lengthy, even after all data has been received, it may be better to let `git` run as much as possible, although not indefinitely.

Reply:
Yeah that's why I've left it at the hammer context.

I guess I could remove that fixme as it seems hammer is the correct context for this.